File: packages/expo-av/android/src/main/java/expo/modules/av/audio/AudioEventHandler.java
package expo.modules.av.audio;

public interface AudioEventHandler {

  void pauseImmediately();

  boolean requiresAudioFocus();

  void updateVolumeMuteAndDuck();

  void handleAudioFocusInterruptionBegan();

  void handleAudioFocusGained();

  void onPause();

  void onResume();
}

File: packages/expo-av/android/src/main/java/expo/modules/av/audio/AudioEventHandler.kt
package expo.modules.av.audio

interface AudioEventHandler {

  fun pauseImmediately()

  fun requiresAudioFocus(): Boolean

  fun updateVolumeMuteAndDuck()

  fun handleAudioFocusInterruptionBegan()

  fun handleAudioFocusGained()

  fun onPause()

  fun onResume()
}
